Microsoft.Alm.Authentication/Git/Installation.cs

```diff
@@ -289,8 +289,8 @@ internal bool IsValid()
 
         public static bool operator ==(Installation install1, Installation install2)
         {
-            return install1._distribution == install2._distribution
-                && PathComparer.Equals(install1._path, install2._path);
+            return install1?._distribution == install2?._distribution
```

Comment:
The correct method here is (and I should have already implemented it this way):

```
if (install1 is null && install2 is null)
    return true;
if (install1 is null || install2 is null)
    return false;

return install1._distribution == install2._distribution
    && PathComparer.Equals(install1._path, install2._path);
```

Reply:
OK... shall I fix my PR or do you do it?

Reply:
I've pushed #587 for your review. Thanks.